core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -1217,7 +1217,9 @@ private Response getObject(final String bucket,
 
           Response.ResponseBuilder res = Response.ok(ris)
               .lastModified(new Date(status.getLastModificationTimeMs()))
-              .header(S3Constants.S3_CONTENT_LENGTH_HEADER, s3Range.getLength(status.getLength()));
+              .header(S3Constants.S3_CONTENT_LENGTH_HEADER, s3Range.getLength(status.getLength()))
+              .header(S3Constants.S3_CONTENT_RANGE_HEADER,
+                  String.format("%s/%s", range, status.getLength()));
```

Comment:
I just want to suggest that we should separate the two ways of Normal reading and Range reading. we need to follow the [Content-Range standard](https://www.rfc-editor.org/rfc/rfc9110.html#name-content-range).
1, if the request is a range reading, the response should include the status code that is 206 and the `Content-Range` header
2, if the request is a normal reading, the status code should be 200 and without the `Content-Range` header.

Reply:
Agreed with the above comment from Jackson.  In addition can you make sure this the response ContentRange format also follow the standard Jackson posted, for range in request param it will be in format of "bytes=1-5" but the response will be "bytes 1-5/<complete_length". And additionally "accept-ranges" header is also needed as specified in aws s3 doc:  https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObject.html#API_GetObject_ResponseSyntax
Since current Alluxio range specification only support Byte Ranges(https://www.rfc-editor.org/rfc/rfc9110.html#name-byte-ranges)  we can just make "accept-ranges" to have the value "bytes"
e.g.
"Content-Range" : "bytes 1-5/1024"
"accept-ranges" : "bytes"

Reply:
fix.